templates/web-app/src/main.tsx

```diff
@@ -0,0 +1,59 @@
+import { StrictMode } from "react";
+import ReactDOM from "react-dom/client";
+import {
+  RouterProvider,
+  createHashHistory,
+  createRouter,
+} from "@tanstack/react-router";
+// Import the generated route tree
+import { routeTree } from "./routeTree.gen";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import reportWebVitals from "internal/reportWebVitals.ts";
+import { AuthManager } from "internal/authManager.ts";
+
+const authManager = new AuthManager(__DOMAIN__, __HABITAT_DOMAIN__, () => {
```

Comment:
Note that having HABITAT_DOMAIN set should be optional. I think we still need it for now because you might want to test on a account hosted on bsky, but still use privi.  If its not set in the env, we should discover the pds url from the did doc of the logged in user.